tests/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# CMake configuration for SHP C++ unit tests
+
+project(${CMAKE_PROJECT_NAME}Tests CXX)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+# Set up GoogleTest
+include(FetchContent)
```

Comment:
as I mentioned previously, could we try to detect if GTest is already present in the system ? A copy & paste from https://github.com/OSGeo/PROJ/blob/master/test/unit/CMakeLists.txt could likely do it

Reply:
Hm, this increase complexity a lot (compared to the three instruction for include(FetchContent), FetchContent_Declare and FetchContent_MakeAvailable). I am also not sure if the link target will be different:  GTest::gtest vs gtest.

So, feel free to go ahead. For me it is good enough as is.

Reply:
ok, I'll possibly follow up on this. Thanks for your work!